internal/testutil/go.mod

```
@@ -0,0 +1,11 @@
+module github.com/elastic/opentelemetry-collector-components/internal/testutil
+
+go 1.22.3
```

Use the latest patched version of this minor version:

```suggestion
go 1.22.12
```